packages/frontend/src/ee/features/aiCopilot/components/AgentChatInput.tsx

```diff
@@ -1,21 +1,35 @@
-import { ActionIcon, Paper, Stack, Textarea } from '@mantine-8/core';
+import {
+    ActionIcon,
+    Avatar,
+    Group,
+    Paper,
+    Stack,
+    Text,
+    Textarea,
+} from '@mantine-8/core';
 import { IconArrowUp } from '@tabler/icons-react';
 import { useLayoutEffect, useRef, useState } from 'react';
+import useUser from '../../../../hooks/user/useUser';
 
 interface AgentChatInputProps {
     onSubmit: (message: string) => void;
     loading?: boolean;
     disabled?: boolean;
+    disabledReason?: string;
 }
 
 export const AgentChatInput = ({
     onSubmit,
     loading = false,
     disabled = false,
+    disabledReason,
 }: AgentChatInputProps) => {
     const inputRef = useRef<HTMLTextAreaElement>(null);
     const [value, setValue] = useState('');
-
+    const user = useUser(true);
+    const name = user.data
+        ? `${user.data.firstName} ${user.data.lastName}`
+        : 'You';
```

Comment:
[sugg] this totally works but I think we tend to use `useApp()` hook for getting the user

Reply:
ah nice! will update